recipe/meta.yaml

```diff
@@ -7,22 +7,36 @@ package:
 source:
   fn: v{{ version }}.tar.gz
   url: https://github.com/gflags/gflags/archive/v{{ version }}.tar.gz
-  sha1: 8bdbade9d041339dc14b4ab426e2354a5af38478
+  sha1: bf07a7926cb2ce4dd4048bdc081f9e00e91cefd6
 
 build:
   number: 0
-  skip: true  # [win]
+  # skipping
+  # py27 (vc9) and py34 (vc10) since vc does not have stdint.h
```

Comment:
This should be solved by adding `msinttypes` as a `build` dependency on Windows particularly for these Pythons.